pandas/_libs/missing.pxd

```diff
@@ -1,6 +1,12 @@
 # -*- coding: utf-8 -*-
 # cython: profile=False
 
-cdef bint is_null_datetimelike(object val)
+from tslibs.nattype cimport is_null_datetimelike
+# pass-through so other modules can cimport is_null_datetimelike from  here
```

Comment:
why is this comment needed?  this is for api simplicity yes? if so i don’t think really necessary for a comment